src/command/add.rs

```diff
@@ -29,46 +29,47 @@ struct IndexEntry {
     path: String,
 }
 
-fn travel_dir(file_name: &String, file_path_list: &mut Vec<String>, hash_list: &mut Vec<String>) {
-    if fs::metadata(file_name).unwrap().is_dir() {
+fn travel_dir(
+    file_name: &String,
+    file_path_list: &mut Vec<String>,
+    hash_list: &mut Vec<String>,
+) -> io::Result<()> {
+    if fs::metadata(file_name)?.is_dir() {
         // 再帰的にaddする
-        for entry in fs::read_dir(file_name).unwrap() {
-            let entry = entry.unwrap();
+        for entry in fs::read_dir(file_name)? {
+            let entry = entry?;
             let path = entry.path();
```

Comment:
僕が書いた時点でこうなってたからこれでいいなら僕が悪いんだけど、これで十分な気がした(なんか問題あるかな)
```suggestion
            let path = entry?.path();
```

Reply:
問題ないです、それにしときます